google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageConfigurationTest.java

```diff
@@ -83,8 +80,12 @@ public void testFromMap() {
 
   @Test
   public void testFromMap_badKey_throwsIae() {
-    thrown.expect(IllegalArgumentException.class);
-    CloudStorageConfiguration.fromMap(ImmutableMap.of("lol", "/omg"));
+    try {
+      CloudStorageConfiguration.fromMap(ImmutableMap.of("lol", "/omg"));
+      Assert.fail();
+    } catch (IllegalArgumentException ex) {
+      assertThat(ex.getClass()).isEqualTo(IllegalArgumentException.class);
```

Comment:
This assert is almost tautological except that it could fail if a subclass of IllegalArgumentException were thrown, which we don't want to fail. It's slightly more useful to assert the message is non-null.

Reply:
done